src/src/PluginStructs/P036_data_struct.cpp

```diff
@@ -284,6 +284,23 @@ bool P036_data_struct::init(taskIndex_t      taskIndex,
   return isInitialized();
 }
 
+String P036_data_struct::ReplaceTokenDelimiters(const String str, const bool ForHeaderOnly = false)
```

Comment:
To make it more lean, you could change `const String str` to `String &str`, so no strings are exchanged via the stack (and an extra copy created), but only a pointer passed, as the intent of these replacements is to be as close to 'in-place' as possible.

Reply:
Done, see last commit